server/routing/admin.py

```diff
@@ -41,20 +41,17 @@ def get_users():
             conn.row_factory = sqlite3.Row
             cur = conn.cursor()
 
-            cur.execute('SELECT * FROM User')
+            cur.execute('SELECT id, username, alerts_email, alerts_phone, isAdmin FROM User')
             rows = cur.fetchall()
 
             users = []
             for row in rows:
                 user1 = {
                     'id': row[0],
                     'username': row[1],
-                    'password': row[2],
-                    'email': row[3],
-                    'phone': row[4],
-                    'alerts_email': row[5],
-                    'alerts_phone': row[6],
```

Comment:
Wondering why the database structure is changed here?

Reply:
> Wondering why the database structure is changed here?

This doesn't change the database structure. The admin dashboard should not show private information such as the email, and phone number of the user.